connector/src/main/scala/com/basho/riak/spark/util/TSConversionUtil.scala

```diff
@@ -69,7 +69,7 @@ object TimeSeriesToSparkSqlConversion {
       case ColumnType.SINT64 => LongType
       case ColumnType.TIMESTAMP => TimestampType
       case ColumnType.VARCHAR => StringType
-      case _ => throw new IllegalStateException("Unsupported column type '" + columnType + "'")
+      case _ => throw new IllegalStateException(s"Unsupported column type $columnType")
```

Comment:
Actually this exception is a bit excess because this matching covers all possible Riak types and default case describes some almost unreal condition